partners/astropy.md

```diff
@@ -0,0 +1,41 @@
+# pyOpenSci's partnership with Astropy
+
+:::{image} /images/astropy_project_logo.svg
+:alt: The Astropy logo. The logo is an orange egg shape with a white snake spiraling in it. The text of the image says The Astropy Project on three lines.
+:width: 600px
+:align: center
+:::
+
+
+pyOpenSci has a collaboration with the [Astropy scientific community](https://www.astropy.org/). Through this collaboration we add an additional layer of peer review checks to Astropy-related packages that go through our [open peer review process](https://www.pyopensci.org/software-peer-review/about/intro.html). If the package meets both the [Astropy criteria for becoming an affiliated package](https://www.astropy.org/affiliated/#becoming-an-affiliated-package) and our pyOpenSci peer review criteria, it can then become both an Astropy-affiliated package and a pyOpenSci ecosystem package.
+
+If you are submitting a package for review and wish to also apply for Astropy affiliation, your package will be:
+
+* Reviewed against current pyOpenSci guidelines and checks.
+* Also reviewed against the Astropy-specific guidelines listed below.
+
+If you are submitting a package for review and wish to also apply for Astropy affiliation, your package will be:
+
+- Reviewed against current pyOpenSci guidelines and checks.
+- Also reviewed against the Astropy-specific guidelines listed below.
```

Comment:
This text is duplicated, just with different symbols ("-" vs "*").

Reply:
oh gosh - good catch. i'll fix that.

Reply:
that must have been a merge conflict fix gone wrong is my only guess 😆